include/ck/tensor_operation/gpu/device/impl/device_batched_gemm_multi_d_xdl.hpp

```diff
@@ -324,8 +324,12 @@ struct DeviceBatchedGemmMultiD_Xdl : public DeviceBatchedGemmMultiD<ALayout,
         index_t BatchStrideE_;
     };
 
+    using ComputeDataType = ADataType;
+
     using GridwiseGemm = GridwiseGemmMultipleD_xdl_cshuffle<
         ADataType, // TODO: distinguish A/B datatype
```

Comment:
Here and in other device op cases A and B datatypes are distinguished, so we can celebrate a massive TODO elimination!